applications/serial_lte_modem/README.rst

```diff
@@ -1,9 +1,9 @@
 .. _serial_lte_modem:
 
-nRF9160: Serial LTE modem
-#########################
+Serial LTE modem
+################
 
-The Serial LTE Modem (SLM) application can be used to emulate a stand-alone LTE modem on the nRF9160.
+The Serial LTE Modem (SLM) application can be used to emulate a stand-alone LTE modem on an nRF91 Series device.
 The application accepts both the modem-specific AT commands documented in the `nRF91 AT Commands Reference Guide <AT Commands Reference Guide_>`_ and proprietary AT commands documented in :ref:`SLM_AT_intro`.
```

Comment:
"nRF91 AT Commands" is no longer correct, the document has been renamed to "nRF9160 AT Commands". Also, there will be a separate AT command document for nRF91x1, but I think the naming has not yet been agreed. I was told the latest proposal (for LTE) was "nRF91x1 Cellular IoT AT commands".

Reply:
Thanks, @tokangas. There are also other places in the NCS documentation where we refer to the nRF91 AT Commands Reference Guide. I will create a separate PR to fix this.

Also, for nRF91x1 (for example, nRF9161), I understand we will have a separate guide for AT commands. So, once the guide is ready and added to the infocenter, I think the documents that refer to the nRF9160 AT command should also have details of the nRF91x1 AT command guide. I will create a Jira task so that this is taken care of.

Reply:
@divipillai I got confirmation from the modem team, that the nRF91x1 AT command guide is named "nRF91x1 Cellular AT Commands". But it's not yet in Infocenter.